app/src/main/res/layout/fragment_splash.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout can contain only the `TextView`

Reply:
Note: I assume you didn't give it much thought since it's mock UI and will 100% be changed in the next couple PRs.
I just wanted to highlight that we have to pay attention when building our layouts to avoid needless view inflation, nesting and additional logic *(i.e. using CL when LL would be good enough)